File: services/objs-service/src/test/java/io/qpointz/poc/objs/service/ObjsServiceAutoConfigurationTest.java
package io.qpointz.poc.objs.service;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class ObjsServiceAutoConfigurationTest {

    @Test
    void shouldInstantiateAutoConfiguration() {
        assertThat(new ObjsServiceAutoConfiguration()).isNotNull();
    }
}

File: objs-service/src/test/kotlin/org/poc/objs/service/ObjsServiceAutoConfigurationTest.kt
package org.poc.objs.service

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class ObjsServiceAutoConfigurationTest {
    @Test
    fun shouldLoadAutoConfigurationClass() {
        assertThat(ObjsServiceAutoConfiguration::class.java).isNotNull()
    }
}
